setup.py

```diff
@@ -17,6 +17,7 @@
     'scikit-learn',
     'tabulate',
     'Lasagne',
+    'pydot'
```

Comment:
You can put this and IPython into the `visualization_require` list below.  This way both should get installed when running tests via Travis.  (That's because Travis installs all extras.)